pages/publishing/amplitude-govern.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 layout: 'guide'
 title: 'Amplitude Govern (API deprecated by Amplitude)'
-# abstract: 'Publishing your tracking plan to Amplitude Govern'
+abstract: 'Publishing your tracking plan to Amplitude Govern'
```

Comment:
Nice! This actually applies to all the sub-pages. Do you want to un-comment those too?